site/content/docs/main/style-guide.md

```diff
@@ -10,55 +10,58 @@ This page outlines writing style guidelines for the Velero documentation and you
 The Velero documentation uses the [kramdown](https://kramdown.gettalong.org/) Markdown renderer.
 
 ## Content best practices
-
-
 ### Use present tense
 
-<table caption="Do and Don't - Use present tense">
-  <tr><th>Do</th><th>Don't</th></tr>
-  <tr><td>This `command` starts a proxy.</td><td>This command will start a proxy.</td></tr>
-</table>
+{{< table caption="Do and Don't - Use present tense" >}}
```

Comment:
this file is missing in the v1.4 docs.

Reply:
Wasn't it new as of the 1.5 cycle?

Reply:
yes, the style guide was new in the 1.5 cycle.